source/sdk/math/Random.ooc

```diff
@@ -39,6 +40,16 @@ Random: class {
 	randFloat: static func -> Float {
 		return (rand() as Float) / INT_MAX
 	}
+	/**
+	   :return: a pseudo-random float number with a normal distribution
+	 */
+	randn: static func (sigma := 1.0f) -> Float {
+		result := 0.0f
+		u1 := randFloat()
+		u2 := randFloat()
+		result = sqrt(-2.0f * log(u1)) * cos(2 * PI * u2)
```

Comment:
Due to a bug in rock , `2 * PI * u2` will always result in an `Int` because the first term, `2`, is an `Int`. Change this to `2.0f * PI * u2`.